agent/src/issuers/mvrc.ts

```diff
@@ -0,0 +1,214 @@
+import { ClaimFormat, JwaSignatureAlgorithm } from '@credo-ts/core'
+import { OpenId4VciCredentialFormatProfile } from '@credo-ts/openid4vc'
+
+import { AGENT_HOST } from '../constants'
+import type {
+  CredentialConfigurationDisplay,
+  MdocConfiguration,
+  PlaygroundIssuerOptions,
+  SdJwtConfiguration,
+} from '../issuer'
+import type { StaticMdocSignInput, StaticSdJwtSignInput } from '../types'
+import { DateOnly, dateToSeconds, oneYearInMilliseconds, serverStartupTimeInMilliseconds } from '../utils/date'
+
+const mvrcDisplay = {
+  locale: 'en',
+  name: 'Vehicle Registration',
+  text_color: '#FFFFFF',
+  background_color: '#CD3401',
+  background_image: {
+    url: '',
+    uri: '',
+  },
+} satisfies CredentialConfigurationDisplay
+
+const mvrcPayload = {
+  issue_date: new DateOnly('2025-01-28'),
+  expiry_date: new DateOnly('2034-08-28'),
+  issuing_country: 'NL',
+  issuing_authority_unicode: 'Fime',
+  document_number: '0123456789',
+}
+
+const mvrcPayload_2 = {
+  issue_date: new DateOnly('2025-01-28'),
+  expiry_date: new DateOnly('2034-08-28'),
+  issuing_country: 'NL',
+  issuing_authority_unicode: 'Fime',
+  document_number: '0123456789',
+  registration_number: '11MM05',
+  date_of_registration: new Date('2021-12-20T17:45:00Z'),
+  date_of_first_registration: '2020-07-14',
+  vehicle_identification_number: 'PD02-5016890',
+  vehicle_holder: [
+    {
+      family_name_unicode: 'baron Van der Cërnosljé',
+      family_name_latin1: 'baron Van der Cërnosljé',
+      given_name_unicode: 'CBA',
+      given_name_latin1: 'CBA',
+      resident_address: '',
+      resident_city: '',
+      resident_country: '',
```

Comment:
I do'nt think we should include empty values? Either omit or add a value

Reply:
Commented them out. so we can optionally add them with values later.